_posts/2020-01-15-Time-To-Run.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Run for SSE Eboard!
+author: SSE Eboard
+tags:
+  - announcement
+  - info
+  - eboard
+hero: ../assets/img/programming_competition/intense.jpg
+overlay: blue
+published: true
+---
+
+{: .lead}
+<!–-break-–>
+
+It's that time of the year again--week 5 of winter quarter. Traditionally, this is one of the hardest times of the year. After all, it's cold outside, we all just got back from a break, midterms are coming (or worse, we're getting grades from the midterms we've already taken), and we're all getting tired of the grind. So why pick this time of the year to ask members to think about running for an eBoard position?
+
+Because this is the time of year when we start thinking that it's time to make a change. We're over halfway through the year here at MSOE, and we probably all have critiques (okay, let's be real, they're probably complaints) by now.
+
+If you're feeling like your software and comp sci classes just aren't preparing you for the future job you want, then it's time to join SSE on the eBoard to help push ideas for supplementing the cirriculum. If you feel like software and computer engineers are just not social enough, and you want the opportunity to meet more of your peers and help them meet each other, then this is the time of year to make a change. If you can think of any way to make life easier or better for any of your software-involved peers, then running for the eBoard is for you. It's time to stop thinking about "I wish..." and time to start thinking "I will."
+
+As an eBoard member you'll have a voice to help make life a little better for your friends and classmates. You'll have the opportunity to meet company representatives as you plan events, gather funding and meet alumni. Not only will you have the priviledge of interacting with these amazing people, but you'll also have the priviledge of harnessing their wants and desires and implementing them here at MSOE, making sure that the next generation of software engineers, computer engineers, computer scientists and their friends are more prepared than ever for the rapidly changing industry (not to mention that knowing some of these people can be a _wonderful_ resource to draw on when you're looking for a job!).
+
+So c'mon, then. Go make a change! Check out the list of eboard [positions](https://docs.google.com/document/d/1I6Dp_j_MUemtdn4tNFSr97WLYEepgRQZpXjQGESEJ2s/edit?usp=sharing)  to see which one is the best fit for you, and fill out a nomination form [here]( https://docs.google.com/document/d/1I6Dp_j_MUemtdn4tNFSr97WLYEepgRQZpXjQGESEJ2s/edit?usp=sharing). Voting will occur on Friday of Week 8, and we look forward to welcoming you aboard!
```

Comment:
The nomination form link is wrong. Currently it's going to the eboard description document. I think the link you want is https://docs.google.com/forms/d/e/1FAIpQLSfJUSC8VcSIqd60WMREZ2iBdwcwgnFJsw3Fvliod9puc3mJig/viewform